dependabot.yml

```diff
@@ -0,0 +1,50 @@
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "lizzcondrey"
+      - "bunchw"
+    labels:
+      - "dependencies"
+    groups:
+      actions:
+        applies-to: version-updates
+        dependency-type: development
+        patterns:
+          - "*"
+
+  
+  - package-ecosystem: "pip"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+    reviewers:
+      - "lizzcondrey"
+      - "bunchw"
+    labels:
+      - "dependencies"
+    groups:
+      runtime_dependencies:
+        applies-to: version-updates
+        dependency-type: production
+        patterns:
+          - "runtime_dependencies"
```

Comment:
We'll need to make sure that our sections in the pyproject.tomls match this group naming. Looking at rest-api, it did not. So, going to add that as a note for our build chain tasks.